### PR TITLE
Upgrade guide: Add section for official npm package

### LINKED ISSUE #150: What was the previous repo of the jquery ui npm package?
The build that has been pushed to NPM is different to previous builds and and breaks various when used with browserify. There is no clear path to upgrade and as jQuery UI doesn't use semver it allows this update breaks when running npm update. 

PS sorry for posting in a seemingly random place - I cant figure where the active community discuss / report issues. IRC was empty.

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes #150

### CODE REVIEW

**Comment by jzaefferer** [comment]
There's some hope that the deamdify issue might get fixed soon: https://github.com/jaredhanson/deamdify/pull/39#issuecomment-235101551

File: page/upgrade-guide/1.12.md [hunk 1]
```diff
@@ -69,6 +69,37 @@ Download builder parses these to resolve each JS component's CSS dependencies. O
 
 We hope to move to a format that resembles more of a standard eventually.
 
+### Official package on npm
+
+jQuery UI 1.12 is the first release officially published to npm as `jquery-ui`. The [previous source repository](https://github.com/devongovett/jqueryui-npm) only covered up to jQuery UI 1.10.5, but never had any of the 1.11 releases. Since the inofficial releases where heavily transformed and fundamentally incompatible with our source files and existing release process, we decided to publish 1.12 under the same name, with different folders and files. Accordingly, the npm releases of 1.12+ are **not backwards compatible** with the 1.10.x releases.
```
Comment (scottgonzalez) [inline page/upgrade-guide/1.12.md]: s/inofficial/unofficial/

**Comment by scottgonzalez** [comment]
Looks good.

**Comment by jzaefferer** [comment]
Fixed the inofficial issue, merged and tagged.

**Pull request merged**